src/utils.ts

```diff
@@ -96,6 +96,9 @@ const isDataPlanSlug = (str: string): boolean =>
 const isStringOrNumber = (value: any): boolean =>
     isString(value) || isNumber(value);
 
+const isEmptyObject = (val: Dictionary<any> | null | undefined): boolean =>
```

Comment:
I originally said to call this `isEmptyObject` because I didn't see the signature when you were sharing your screen, but it seems like you're checking for arrays here as well (and in your tests too).  If that's the case, maybe reverting to `isEmpty` is more accurate.  Was this function signature pulled from somewhere (and if so, a comment with the original source would be useful like we do throughout [polyfill.js](https://github.com/mParticle/mparticle-web-sdk/blob/master/src/polyfill.js))

Reply:
Nope. I'm completely to blame for this.